module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -364,8 +364,24 @@ protected function getSearchResultsView($setupCallback = null)
         if (isset($view->params) && $view->params->getView() == 'rss') {
             $response = $this->getResponse();
             $response->getHeaders()->addHeaderLine('Content-type', 'text/xml');
-            $feed = $this->getViewRenderer()->plugin('resultfeed');
-            $response->setContent($feed($view->results)->export('rss'));
+            $feedHelper = $this->getViewRenderer()->plugin('resultfeed');
+            $feed = $feedHelper($view->results);
+            $writer = new \Laminas\Feed\Writer\Renderer\Feed\Rss($feed);
+            $writer->render();
+            $themeInfo = $this->serviceLocator->get(\VuFindTheme\ThemeInfo::class);
+            $themeHits = $themeInfo->findInThemes('assets/xsl/rss.xsl');
+            if (count($themeHits) > 0) {
```

Comment:
Wouldn't `if ($themeHits) {` suffice?

Reply:
Fixed as part of [060f1e9](https://github.com/vufind-org/vufind/pull/2385/commits/060f1e9eb85fed9a782aa22c8e3b3a50938b8c65).